cysync/src/pages/mainApp/sidebar/swap/panels/dialogs/received.tsx

```diff
@@ -0,0 +1,161 @@
+import { CallReceived, ContentCopy } from '@mui/icons-material';
```

Comment:
I don't think we need a `panels` folder. It can just be `swap/dialog` `swap/exchange` etc